threeplay.go

```diff
@@ -25,17 +25,11 @@ const (
 	HTML OutputFormat = "html"
 )
 
-// HTTPClient interface for http clients used in the 3Play client
-type HTTPClient interface {
-	Get(string) (*http.Response, error)
-	PostForm(string, url.Values) (*http.Response, error)
-}
-
 // Client 3Play Media API client
 type Client struct {
-	apiKey    string
-	apiSecret string
-	client    HTTPClient
+	apiKey     string
+	apiSecret  string
+	HTTPClient *http.Client
```

Comment:
Does gock patch the DefaultClient? Are we using the DefaultClient instance?

I don't like using the default client in a library because it's a global, and other libraries can temper with it.